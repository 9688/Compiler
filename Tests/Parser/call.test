"a" > a || a == 'a';
b >= c && 0xF || 1.0;