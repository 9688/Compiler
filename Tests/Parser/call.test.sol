"ROOT"
    "EXPR"
         "||"
              ">"
                   "a"
                   "a"
              "=="
                   "a"
                   "a"
    "EXPR"
         "||"
              "&&"
                   ">="
                        "b"
                        "c"
                   "15"
              "1"
