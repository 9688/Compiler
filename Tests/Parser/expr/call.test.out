
list
     "||"
          ">"
               a
               a
          "=="
               a
               a
list
     "||"
          "&&"
               ">="
                    b
                    c
               15
          1

